routes.mjs

```diff
@@ -0,0 +1,14 @@
+import { resolve } from 'path';
+import db from './db/models/index.mjs';
+import initUsersController from './controllers/users-controller.mjs';
+
+export default function routes(app) {
+  const usersController = initUsersController(db);
+
+  app.post('/register', usersController.signup);
+  app.post('/login', usersController.login);
+  app.get('/verify-cookie', usersController.verify);
+  app.get('*', (request, response) => {
+    response.sendFile(resolve('dist', 'main.html'));
+  });
```

Comment:
I think backend won't need this route because this one was mainly for frontend react router? I might be wrong

Reply:
On a side note can we have all backend endpoints to be prefixed with /api so we will know that its a backend route vs a react route

eg. `/api/login`

Reply:
okay!